### PR TITLE
Added recipe for csvkit

### PR DESCRIPTION
**Pull request by pmlandwehr**
This recipe for `csvkit` ticks the version above 1.0 (contra the base version in the `anaconda` channel), replacing most of the old dependencies with `agate` and affiliated libraries.

### CODE REVIEW

**Comment by pmlandwehr** [comment]
Missing a `py36` build for `agate-excel`. Fixing...

**Comment by pmlandwehr** [comment]
Feeling stumped by this error message. Any thoughts, @ocefpaf ?

File: recipes/mesalib/build.sh [hunk 1]
```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
```
Comment (jakirkham) [inline recipes/mesalib/build.sh]: Please don't delete the other recipes.

**Comment by jakirkham** [comment]
Please rebase on `master`.

**Pull request closed without merging**